apps/demos/testing/widgets/datagrid/EditStateManagement.test.js

```diff
@@ -14,7 +14,7 @@ runManualTest('DataGrid', 'EditStateManagement', ['jQuery', 'React', 'Vue', 'Ang
     const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
 
     // edit row
-    await t.click($('a').withText('Edit').nth(0));
+    await t.click($('.dx-link-edit').nth(0));
```

Comment:
Usually we use constant variables for class names to avoid duplication. But, these tests belong to the grid squad, let them decide if it's important for them or not

Reply:
We need to use our testcafe models here, we will refactor it when we'll have time (not soon)